modules/client/src/client-common/interfaces/context.ts

```diff
@@ -1,26 +1,31 @@
 // This file defines the interfaces of the context object holding client settings
 
 import { Signer } from "@ethersproject/abstract-signer";
-import { JsonRpcProvider, Networkish } from "@ethersproject/providers";
+import { JsonRpcProvider, Network, Networkish } from "@ethersproject/providers";
 import { Client as IpfsClient } from "@aragon/sdk-ipfs";
 import { GraphQLClient } from "graphql-request";
 
 // Context input parameters
 
 type Web3ContextParams = {
-  network: Networkish;
-  signer?: Signer;
+  // Network should be provided
```

Comment:
Could we use these types of comments for struct fields, so that everyone gets code completion with tooltips for free when developing?

```suggestion
  /** Network should be provided */
```